python/res/job_queue/job_queue_manager.py

```diff
@@ -145,16 +124,28 @@ def __repr__(self):
     def max_running(self):
         return self.get_job_queue().get_max_running()
 
+    def max_submit(self):
+        return self.get_job_queue().get_max_submit()
+
     def execute_queue(self):
 
         job_queue = self.get_job_queue()
         started_job_threads = []
         while job_queue.is_running():
             job = job_queue.fetch_next_waiting()
             while not job_queue.stopped_by_user and job is not None and job_queue.count_running() <= self.max_running():
```

Comment:
Doesn't this add one job more than it should?

Reply:
I think you are right, however this is outside the scope of the PR. Well spotted though :clap: Make an issue ;)

Reply:
yes, as was pointed out by @markusdregi it's outside this scope; mainly we count the run number from `1`